src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN yum upgrade -y && \
       xz \
       zlib-devel.i686 \
       zlib-devel.x86_64 \
+      jemalloc-devel \
```

Comment:
You don't need `-devel` -- the base package includes a `jemalloc.sh` wrapper script that will set `LD_PRELOAD` for you.

Reply:
Sorry for the delay, do you know where this jemalloc.sh script is located?

Reply:
It's `/usr/bin/jemalloc.sh`, so that's even in the `PATH`.